EarlGrey.xcodeproj/project.pbxproj

```diff
@@ -1286,6 +1286,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = com.google.earlgrey.EarlGrey;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = NO;
+				SWIFT_INSTALL_OBJC_HEADER = NO;
```

Comment:
To confirm, this basically stops the Objc Compatibility Header from being created, right? Any reason why we would want that?

Reply:
mentioned here: https://github.com/google/EarlGrey/issues/636#issuecomment-334890424